### PR TITLE
fix(argo-rollouts): add CI for enabling extra objects for argo-rollouts

### PR DESCRIPTION
**Pull request by jmeridth**
Currently passing. Would expect it to fail until #2196 merges. 

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [ ] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [ ] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

File: charts/argo-rollouts/ci/enable-extra-objects.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Do not deploy the CRDs as they are already present from the previous test
+installCRDs: false
+
+extraObjects:
+  - apiVersion: v1
+    kind: Secret
+    metadata:
+      name: datadog
+    type: Opaque
+    data:
```
Comment (mkilchhofer) [inline charts/argo-rollouts/ci/enable-extra-objects.yaml:11]: I think you need to use `stringData` here or the values of your dummy data needs to be base64 encoded

Comment (jmeridth) [reply charts/argo-rollouts/ci/enable-extra-objects.yaml:11]: @mkilchhofer you nailed it.  Thank you.

Comment (mkilchhofer) [reply charts/argo-rollouts/ci/enable-extra-objects.yaml:11]: Perfect. Now you need to rename the file. When CI should pick it up, it needs to end with `...-values.yaml` :)

**Review by mkilchhofer — APPROVED** [review]

**Comment by jmeridth** [comment]
[Lint success](https://github.com/argoproj/argo-helm/actions/runs/5799188976/job/15718627701?pr=2198#step:7:40)

[Tests pass with new values file](https://github.com/argoproj/argo-helm/actions/runs/5799188976/job/15718627701?pr=2198#step:7:58)

**Review by yu-croco — APPROVED** [review]

**Pull request merged**